test/functionalTest/viewDiffs.sh

```diff
@@ -0,0 +1,54 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Good idea.
This way you can start a 'make test' without CB_DIFF_TOOL and go take a shower.
When you get back you can view all the diffs.